main.tf

```diff
@@ -1,3 +1,8 @@
+provider "aws" {
+  region = "us-east-1"
+  alias = "us-east-1"
+}
```

Comment:
I think Terraform recommends not putting providers within modules:

https://www.terraform.io/docs/language/modules/develop/providers.html

Reply:
Oh right, that makes sense!

I've changed it to a [proxy configuration block](https://www.terraform.io/docs/language/modules/develop/providers.html#proxy-configuration-blocks) and updated the test and example. You have to pass in the provider even if the region is already `us-east-1`, but I can't find a better way to do it 😞

Reply:
damn this stinks. I'm not a fan of requiring a new input essentially, but I too can't find a better way to allow all resources use the root provider while just having certs in us-east-1. 🤷